types/alexa-voice-service/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Type definitions for alexa-voice-service 0.0
-// Project: https://github.com/miguelmota/alexa-voice-service.js
```

Comment:
Why was this changed? 

https://github.com/miguelmota/alexa-voice-service.js is the actual URL

Reply:
It's listed without `.js` in their package.json, seems like the repo was renamed since without it redirects to the new one. Submitted a PR here: https://github.com/miguelmota/alexa-voice-service.js/pull/15

Reply:
@dolanmiu when the urls are very similar and one redirects to the other, I kept the one from package.json because it's the source of truth, even when (1) not https or (2) missing a small name change. @LinusU has the right idea to fix it upstream.